voice-agent-tester/README.md

```diff
@@ -0,0 +1,57 @@
+# Samples for using the microservice
+
+This directory contains samples for understanding and using the [Voice Agent Tester](https://www.ibm.com/support/knowledgecenter/SS4U29/VAtesterMS_overview.html) microservice. You can quickly start using the microservice by using the `docker-compose.yml` which includes the Voice Agent Tester, a SIP Orchestrator and a Media Relay microservices which are the components for the Caller Voice Gateway, and a CouchDB for the test persistence. Just fill in the variables and run `docker-compose up`. If you're going to use the CouchDB service as your persistence then you would have to run a [first time setup](https://docs.couchdb.org/en/master/setup/index.html#setup) and set the CLOUDANT_URL variable pointing to the url on which CouchDB is exposed like http://localhost:5984, also set CLOUDANT_USERNAME and CLOUDANT_PASSWORD to the appropiate values that you added for authentication to the database.
+
```

Comment:
For a list of configuration items and their use you can click [here](https://www.ibm.com/support/knowledgecenter/SS4U29/VAtesterMS_environment_var.html)